chromium_src/ui/gfx/color_palette.h

```diff
@@ -10,13 +10,10 @@
 
 namespace gfx {
 
-constexpr SkColor kBraveGrey700 = SkColorSetRGB(0x4A, 0x4A, 0x4A);
-constexpr SkColor kBraveGrey800 = SkColorSetRGB(0x3b, 0x3e, 0x4f);
-constexpr SkColor kBraveNeutral300 = SkColorSetRGB(0xDE, 0xE2, 0xE6);
-constexpr SkColor kBraveNeutral800 = SkColorSetRGB(0x34, 0x3A, 0x40);
-constexpr SkColor kBraveBlurple300 = SkColorSetRGB(0xA0, 0xA5, 0xEB);
-constexpr SkColor kBraveColorBrand = SkColorSetRGB(0xfb, 0x54, 0x2b);
-constexpr SkColor kBraveColorOrange300 = SkColorSetRGB(0xFF, 0x97, 0x7D);
+// TODO(simonhong): Remove this. kBraveBlurple300 is used incorrectly.
+inline constexpr SkColor kBraveBlurple300 = SkColorSetRGB(0xA0, 0xA5, 0xEB);
```

Comment:
Optional: use Nala colors. The names should match what's in the Figma files

Reply:
In the interest of keeping this PR as minimal as possible, I'm happy to land this as-is.